ibc_public/utils_data.py

```diff
@@ -41,17 +41,35 @@
                                      CONTRASTS['right label'][i]]
 
 
-def get_subject_session(protocol):
-    """ utility to get all (subject, session) for a given protocol"""
+def get_subject_session(protocols: list) -> list:
```

Comment:
`def get_subject_session(protocols):`
types are given in the dostring and only there. The code has to follow the numpy documentation conventions

https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
Done